'round' is not a member of 'std' on Ubuntu 14.04

Compiling `example/stm32f4_discovery/blink` inside the xpcc-vm leads to the following compile error:
```
In file included from /home/rca/xpcc/build/stm32f4_discovery/blink/libxpcc/src/xpcc/architecture/../../../generated_platform/drivers.hpp:67:0,
                 from /home/rca/xpcc/build/stm32f4_discovery/blink/libxpcc/src/xpcc/architecture/platform.hpp:25,
                 from ../stm32f4_discovery.hpp:19,
                 from main.cpp:1:
/home/rca/xpcc/build/stm32f4_discovery/blink/libxpcc/src/xpcc/architecture/../../../generated_platform/driver/clock/stm32/static.hpp: In static member function 'static xpcc::clock::StartupError xpcc::stm32::PllSetup<InputFrequency, OutputFrequency, UsbFrequency, Source>::enable()':
/home/rca/xpcc/build/stm32f4_discovery/blink/libxpcc/src/xpcc/architecture/../../../generated_platform/driver/clock/stm32/static.hpp:67:32: error: 'round' is not a member of 'std'
     xpcc::clock::ns_per_loop = std::round(1000000000.f / OutputFrequency * 3.f);
                                ^
/home/rca/xpcc/build/stm32f4_discovery/blink/libxpcc/src/xpcc/architecture/../../../generated_platform/driver/clock/stm32/static.hpp:67:32: note: suggested alternative:
In file included from /usr/arm-none-eabi/include/c++/4.9.3/cmath:44:0,
                 from /home/rca/xpcc/src/xpcc/math/filter/pid.hpp:35,
                 from /home/rca/xpcc/src/xpcc/math/filter.hpp:41,
                 from /home/rca/xpcc/src/xpcc/math.hpp:39,
                 from /home/rca/xpcc/src/xpcc/math/tolerance.hpp:14,
                 from /home/rca/xpcc/src/xpcc/architecture/interface.hpp:15,
                 from /home/rca/xpcc/build/stm32f4_discovery/blink/libxpcc/src/xpcc/architecture/../../../generated_platform/driver/can/generic/can_bit_timings.hpp:13,
                 from /home/rca/xpcc/build/stm32f4_discovery/blink/libxpcc/src/xpcc/architecture/../../../generated_platform/drivers.hpp:36,
                 from /home/rca/xpcc/build/stm32f4_discovery/blink/libxpcc/src/xpcc/architecture/platform.hpp:25,
                 from ../stm32f4_discovery.hpp:19,
                 from main.cpp:1:
/usr/arm-none-eabi/include/math.h:278:15: note:   'round'
 extern double round _PARAMS((double));
```

Compiler version is 4.9.3.

--- a/examples/stm32f072_discovery/stm32f072_discovery.hpp
+++ b/examples/stm32f072_discovery/stm32f072_discovery.hpp
@@ -49,7 +49,7 @@ struct systemClock
 		xpcc::clock::fcpu     = Frequency;
 		xpcc::clock::fcpu_kHz = Frequency / 1000;
 		xpcc::clock::fcpu_MHz = Frequency / 1000000;
-		xpcc::clock::ns_per_loop = std::round(4000 / (Frequency / 1000000));
+		xpcc::clock::ns_per_loop = ::round(4000 / (Frequency / 1000000));
 
 		return true;
 	}

--- a/examples/stm32f7_discovery/stm32f7_discovery.hpp
+++ b/examples/stm32f7_discovery/stm32f7_discovery.hpp
@@ -94,7 +94,7 @@ struct systemClock
 		xpcc::clock::fcpu     = Frequency;
 		xpcc::clock::fcpu_kHz = Frequency / 1000;
 		xpcc::clock::fcpu_MHz = Frequency / 1000000;
-		xpcc::clock::ns_per_loop = std::round(1000 / (Frequency / 1000000));
+		xpcc::clock::ns_per_loop = ::round(1000 / (Frequency / 1000000));
 
 		return true;
 	}

--- a/src/xpcc/architecture/platform/driver/clock/lpc/static.hpp.in
+++ b/src/xpcc/architecture/platform/driver/clock/lpc/static.hpp.in
@@ -62,7 +62,7 @@ namespace xpcc
 				xpcc::clock::fcpu     = OutputFrequency;
 				xpcc::clock::fcpu_kHz = OutputFrequency / 1000;
 				xpcc::clock::fcpu_MHz = OutputFrequency / 1000000;
-				xpcc::clock::ns_per_loop = std::round(1000000000.f / OutputFrequency * 4.f);
+				xpcc::clock::ns_per_loop = ::round(1000000000.f / OutputFrequency * 4.f);
 				return StartupError::None;
 			}
 		};

--- a/src/xpcc/architecture/platform/driver/clock/stm32/static.hpp.in
+++ b/src/xpcc/architecture/platform/driver/clock/stm32/static.hpp.in
@@ -70,7 +70,7 @@ namespace xpcc
 				xpcc::clock::fcpu     = OutputFrequency;
 				xpcc::clock::fcpu_kHz = OutputFrequency / 1000;
 				xpcc::clock::fcpu_MHz = OutputFrequency / 1000000;
-				xpcc::clock::ns_per_loop = std::round(1000000000.f / OutputFrequency * 3.f);
+				xpcc::clock::ns_per_loop = ::round(1000000000.f / OutputFrequency * 3.f);
 				return StartupError::None;
 			}
 		};
@@ -114,7 +114,7 @@ namespace xpcc
 				xpcc::clock::fcpu     = OutputFrequency;
 				xpcc::clock::fcpu_kHz = OutputFrequency / 1000;
 				xpcc::clock::fcpu_MHz = OutputFrequency / 1000000;
-				xpcc::clock::ns_per_loop = std::round(1000000000.f / OutputFrequency * 3.f);
+				xpcc::clock::ns_per_loop = ::round(1000000000.f / OutputFrequency * 3.f);
 				return StartupError::None;
 			}
 		};

